jdaviz/configs/imviz/tests/test_catalogs.py

```diff
@@ -250,11 +250,6 @@ def test_offline_ecsv_catalog(imviz_helper, image_2d_wcs):
     assert len(imviz_helper.app.data_collection) == 2  # image + markers
 
     catalogs_plugin.clear_table()
-
-    assert not catalogs_plugin._obj.results_available
-    assert len(imviz_helper.app.data_collection) == 2  # markers still there, just hidden
```

Comment:
What does this removal mean? We cannot just hide the markers anymore?

Reply:
right, it wasn't easy to keep this with the new callback mechanism (again, needed since we now want multiple tables)... but I can work to re-implement if this is something we want to keep.

Reply:
Gotcha. I am gonna link https://github.com/spacetelescope/jdaviz/pull/3429#discussion_r1956436961 here so others don't have to track this convo in 2 different places.